pkg/overlay/cache.go

```diff
@@ -115,17 +109,20 @@ func (o *Cache) Put(nodeID string, value pb.Node) error {
 	if err != nil {
 		return err
 	}
-
 	return o.DB.Put(node.IDFromString(nodeID).Bytes(), data)
 }
 
 // Bootstrap walks the initialized network and populates the cache
 func (o *Cache) Bootstrap(ctx context.Context) error {
-	nodes, err := o.DHT.GetNodes(ctx, "", 1280)
+	r, err := randomID()
```

Comment:
why did we change from empty string to randomID?

Reply:
I don't understand any scenario where empty string is ideal.  Am I missing something?

Reply:
Reading Dennis's new comments he added which describe what this function is supposed to do, I do believe empty string is best.

Reply:
Hrmm, I guess that I misunderstood the DHT.GetNodes interface.  Reverting to empty string.